### DISCUSSION TITLE
New Datasets

### DISCUSSION THREAD

**Discussion by pykt-team** [pykt-team/pykt-toolkit#29] [Ideas]
If you want to add a new dataset, you can discuss it here.

**Comment by Doralene** [comment]
‘Junyi’ is also a widely used dataset. How about adding this dataset to your project? This dataset has two versions. One was released in 2015, you can find it on pslcdatashop. The other one was shared in 2020, you can find it on Kaggle.

**Reply by sonyawong** [reply]
> ‘Junyi’ is also a widely used dataset. How about adding this dataset to your project? This dataset has two versions. One was released in 2015, you can find it on pslcdatashop. The other one was shared in 2020, you can find it on Kaggle.

Thank you for your attention, we will try to add Junyi into pyKT in the future.
